_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
+
+- Managed-service backups: Cockroach Labs takes automated backups of **{{ site.data.products.serverless }}** and **{{ site.data.products.dedicated }}** clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for both {{ site.data.products.db }} clusters.
+- {% if page.name == "run-bulk-operations.md" %} Customer-owned backups {% else %} [Customer-owned backups](../cockroachcloud/run-bulk-operations.html) {% endif %}: You can take manual backups and store them in your [cloud storage buckets](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html) using the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) statement. 
```

Comment:
```suggestion
- {% if page.name == "run-bulk-operations.md" %} **Customer-owned backups**: {% else %} [**Customer-owned backups**](../cockroachcloud/run-bulk-operations.html): {% endif %} You can take manual backups and store them in your [cloud storage buckets](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html) using the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) statement. 
```

Reply:
Also, double check this in the preview once you have merged this suggestion (or test), but there was a space between "Customer-owned backup" and the colon... I think this was due to the placement of the colon within the if/else block